ipam/ippool_manager_test.go

```diff
@@ -416,7 +422,7 @@ var _ = Describe("IPPool manager", func() {
 					},
 					Status: ipamv1.IPClaimStatus{
 						Address: &corev1.ObjectReference{
-							Name:      "abcpref-192-168-1-12",
+							Name:      "abcpref-192-168-1-10",
```

Comment:
Would you please check if this is correct? I think it doesn't need this change.

Reply:
yes, thank you for noticing. Fixed it.